magda-web-client/src/Components/Dataset/View/DatasetPageSuggestForm.js

```diff
@@ -83,29 +83,47 @@ export default class DatasetPageSuggestForm extends React.Component {
                 maxHeight: "95vh"
             }
         };
+        let contactPointMatch;
+        if (
+            this.props?.contactPoint &&
+            typeof this.props?.contactPoint === "string"
+        ) {
+            contactPointMatch = this.props.contactPoint.match(
```

Comment:
How about doing this? Current regex missed `asd_ssd@dmail.com` etc and unnecessarily captured whitespace. 
Plus, we already used regex no need to manually process the string 
```
const emailRegex = /[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}/ig;
const matches = this.props.contactPoint.match(emailRegex);
contactPointMatch = matches?.length ? matches[0] : this.props.contactPoint;
```

Reply:
Looks good! Thanks, fixed.